transmogrifier/sources/json/aardvark.py

```diff
@@ -86,19 +88,106 @@ def get_optional_fields(self, source_record: dict) -> dict | None:
 
+    @staticmethod
+    def get_alternate_titles(source_record: dict) -> list[timdex.AlternateTitle]:
+        """Get values from source record for TIMDEX alternate_titles field."""
+        alternate_titles = []
+
+        if "dct_alternative_sm" in source_record:
+            for title_value in [
+                title_value for title_value in source_record["dct_alternative_sm"]
+            ]:
+                alternate_titles.append(timdex.AlternateTitle(value=title_value))
+
+        return alternate_titles
+
+    @staticmethod
+    def get_contributors(source_record: dict) -> list[timdex.Contributor]:
+        """Get values from source record for TIMDEX contributors field."""
+        contributors = []
+
+        if "dct_creator_sm" in source_record:
+            for contributor_value in [
+                contributor_value
+                for contributor_value in source_record["dct_creator_sm"]
+            ]:
+                contributors.append(
+                    timdex.Contributor(value=contributor_value, kind="Creator")
+                )
+
+        return contributors
+
+    @staticmethod
+    def get_notes(source_record: dict) -> list[timdex.Note]:
+        """Get values from source record for TIMDEX notes field."""
+        notes = []
+
+        if "gbl_displayNote_sm" in source_record:
+            for note_value in [
+                note_value for note_value in source_record["gbl_displayNote_sm"]
+            ]:
+                notes.append(timdex.Note(value=[note_value], kind="Display note"))
+
+        return notes
+
+    @staticmethod
+    def get_publication_information(source_record: dict) -> list[str]:
+        """Get values from source record for TIMDEX publication_information field."""
+        publication_information = []
+
+        if "dct_publisher_sm" in source_record:
+            publication_information.extend(source_record["dct_publisher_sm"])
+
+        if "schema_provider_s" in source_record:
+            publication_information.append(source_record["schema_provider_s"])
+
+        return publication_information
+
+    @staticmethod
+    def get_rights(source_record: dict) -> list[timdex.Rights]:
+        """Get values from source record for TIMDEX rights field."""
+        rights = []
+
+        if "dct_accessRights_s" in source_record:
+            rights.append(
+                timdex.Rights(
+                    description=source_record["dct_accessRights_s"], kind="Access"
+                )
+            )
+
+        if "dct_license_sm" in source_record:
+            rights.append(timdex.Rights(uri=source_record["dct_license_sm"]))
+
+        for aardvark_rights_field in ["dct_rights_sm", "dct_rightsHolder_sm"]:
+            if aardvark_rights_field in source_record:
+                rights.append(
+                    timdex.Rights(
+                        description=". ".join(source_record[aardvark_rights_field])
+                    )
+                )
+
+        return rights
```

Comment:
I like this too.  As mentioned in my own PR about a second "QA" pass on these transformations, these are clear and easy to follow.  If we decide that things should be tweaked, there isn't much logic to untangle; just move things around as needed.